tortoise/queryset.py

```diff
@@ -716,6 +716,9 @@ def resolve_to_python_value(self, model: "Type[Model]", field: str) -> Callable:
             # return as is to get whole model objects
             return lambda x: x
 
+        if field in [x[1] for x in model._meta.db_native_fields]:
```

Comment:
extremely minor and probably won't make much impact but isn't a generator technically faster since we're only iterating once and thus stop generating values once the field has been found?

Reply:
This function returns a mapping of callables, it is run once per model typically, so the perf improvement was returning a identity function for fields that are natively supoorted by the DB.
So the perf here is not a big deal.

Reply:
ah my bad, figured it was a performance hotpot, still getting to know the code, sorry

Reply:
No worries, I have learnt to not guess hot-spots any more as I have a history to be wildly wrong.
Definitely use a profiler to see what is going on.

I actually did a talk about this a few years ago.
Step 1, get some kind of repeatable benchmark going (to verify what you are doing moves the needle in the right direction)
Step 2, use a profiler as non-obvious things appear, like "why do we prepare a SQL statement for every call?" Or that the `cursor.DictCursor` class was spending a lot of time building dicts.

Reply:
yeah my guess was mostly based on the fact that it changed here instead of a profiler :P